databricks_sync/sdk/generators/identity.py

```diff
@@ -263,7 +269,6 @@ def _user_group_has_profile(self, user, role_arn):
                 return True
         except Exception:
             log.exception("Failed to check if user group has profile")
```

Comment:
```suggestion
        except Exception as e:
            log.exception("Failed to check if user group has profile with error {str(e)}")
```

Should probably catch the error message in exception here too.